collector.proto

```diff
@@ -34,7 +44,7 @@ message KeyValue {
 
 message Log {
     google.protobuf.Timestamp timestamp = 1;
-    repeated KeyValue keyvalues = 2;
+    repeated LogField fields = 2;
```

Comment:
I can see this either way. My very slight bias would be to only have one "key-value-like" type rather than the two presented here, just in the name of reducing the number of concepts. But whatever.